pkg/validation/validation.go

```diff
@@ -87,6 +87,38 @@ func validateRegSync(release string) error {
 	// RKE2 and K3s releases may need to be fixed after the fact,
 	// so we just make sure we don't remove any released image or tag
 	for name, tags := range upstream {
+		excluded := map[string]bool{
```

Comment:
Could you comment explaining the rationale behind this and that will be cleaned up after release-v2.12 is created?